dev-libs/elftoolchain/elftoolchain-0.7.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Libraries/utilities to handle ELF objects (BSD drop in replacement for libelf)"
+HOMEPAGE="https://wiki.freebsd.org/LibElf"
+SRC_URI="https://netcologne.dl.sourceforge.net/project/elftoolchain/Sources/${P}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-arch/libarchive
+	dev-libs/expat
+	dev-libs/libbsd
+	!dev-libs/elfutils
+	!dev-libs/libelf"
+DEPEND="${RDEPEND}
+	virtual/yacc"
```

Comment:
this is used on `CHOST`?

Reply:
Oh good point, this was a quick draft of the ebuild, I'll make small fixups, but I'm fairly certain we don't need yacc on CHOST